View item from list fragment

tlmurphy opened the pull request:
Still more work that needs to be done, specifically re-working the SearchableItem classes into a more MVP like structure. There's also some left over UI elements (like a floating action button in the favorites activity) that I need to remove.

![ezgif com-resize 2](https://user-images.githubusercontent.com/12003501/39105573-5ad66c10-467c-11e8-8c40-52051e46c045.gif)

jmhossler commented:
I'm confused; this seems to be going against the flow that Pete put out earlier with the list fragment. Could you explain the overall idea of this structure you created?

tlmurphy commented:
The item list fragment is pretty much the same, just changed the ui and added it to where we can select an item from the list. The dummy stuff seemed handy, it came from the template but we’ll remove it eventually. 

The only thing that is off to me is the current implementation of the searchable item fragment/activity which is just taken from the template and needs to be re worked.

tlmurphy commented:
@padietl @delennsapp if you guys need this updated code for what you're doing, I could make a new pr for updating the structure of Searchabltem and we could merge this in.
